### PR TITLE
Add github actions to implement CI, remove STDCI support

### PR DESCRIPTION
**Pull request by sjd78**
#### GitHub Actions to implement CI changes
  - Two parallel sets of jobs run.  One set test then build using yarn offline / ovirt-engine-nodejs-modules.  The other set test and build using yarn online / direct download of dependencies.

  - If the test job fails, the build job will not run.

  - Only the offline built rpms are exported.

  - Add `rpmlint` to the build to verify the spec file (especially the changelog dates).

  - Refactored the build scripts to build in `$repo_root/tmp.repos`.

  - Added exclusions / ignores as appropriate to `tmp.repos/`.

  - Set `.eslintrc.js` to be the root config.  If this is not done, eslint will look in every parent for another config and cause annoying problems when the rpm is being built.

#### Remove STDCI support changes
  - Remove `automation.yaml` and related `automation/*.packages`, `automation/*.repos` and `automation/*.sh` files.  Without the base yaml file, std-ci won't attempt to run anything.

  - Github actions take over PR CI testing and builds

  - copr provides builds from the master branch (https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/)

  - `automation/build.sh` is retained with specific accommodations for std-ci/jenkins package caching issues removed

#### CI build with oVirt buildcontainer

  - To minimize the container setup on the base `centos:stream8` container, use the `quay.io/ovirt/buildcontainer:el8stream` container.
  
  - Update the spec `BuildRequires` to include the requirements for source build scripts.  These will be picked up by the ovirt/buildcontainer and we will not need to explicitly install them in the github workflows using the container.

  - **NOTE**: For the webpack build jobs to work, nodejs needs to be at least v14.  https://github.com/oVirt/buildcontainer/pull/4 enables nodejs:14 on the el8 build.  ~~That PR needs to be merged before the jobs here will succeed.~~ The PR has been merged.

#### Update copr Makefile to match spec BuildRequires

  - Align the copr build dependency installs with the spec file `BuildRequires`.

### EL9 builds with webpack 4

***A note about centos:stream9 / el9 builds with webpack 4***

While working on this PR, I found out that any project using a webpack version less than 5.61.0 cannot build on el9. There was a change to openssl and nodejs that broke a hash function used by webpack.  Initially the break was localized to node 17, but the current nodejs 16.10.0 build on centos stream 9 has the same issue AND the command line option workaround is blocked.  See https://github.com/webpack/webpack/issues/14532 for more details.  

The only way for our projects to build on el9 is to upgrade to webpack ^5.61.0.

### CODE REVIEW

**Comment by sjd78** [comment]
@sgratch , @rszwajko - Everything is working as it should! I'm running a [copr build on my project](https://copr.fedorainfracloud.org/coprs/sdickers/test_ovirt_builds/build/3363985/) to verify those changes.

File: packaging/spec.in [hunk 1]
```diff
@@ -32,6 +38,7 @@ Extensions include:
   - Cluster upgrade wizard
   - Host copy network dialog
   - VM export dialog
+  - VM CPU host pinning dialog
```
Comment (michalskrivanek) [inline packaging/spec.in:41]: unrelated?

**Review by michalskrivanek — APPROVED** [review]
lgtm

File: packaging/spec.in (hunk 1 above)
Comment (sjd78) [reply packaging/spec.in]: true - I'll leave it for the new release update
